src/components/Global/Header/index.tsx

```diff
@@ -228,7 +228,12 @@ const MenuLinks = () => {
             <NavLink href={'/send'} onClick={handleClick}>
                 <Text display="block">send</Text>
             </NavLink>
-            <NavLink href={'/request/create'}>
+            <NavLink
+                href={'/request/create'}
+                onClick={() => {
+                    if (window.location.pathname == '/request/create') window.location.reload()
+                }}
+            >
```

Comment:
TODO: nice - I think this is the desired behavior. For consistency, can we have the same behaviour across all pages? So send, request, cashout.

Generally when submitting a PR we should think holistically - else we get fragmented behaviour. Even though the ticket was submitted only for request links navbar, its valid across other items, so the issue fix shouldn't just tackle the complaint, but ideally tackle whole issue holistically.  And if not, it should at least make mention of this wider issue

a very similar thing was raised in this PR from @panosfilianos: https://github.com/peanutprotocol/peanut-ui/pull/426#pullrequestreview-2356706285